siswrap/wrapper_services.py

```diff
@@ -174,8 +200,7 @@ def run(self):
                 proc = subprocess.Popen(["/bin/sleep", "1m"])
                 exec_string = "/bin/sleep 1m"
             else:
-                exec_string = ExecString(self, self.conf_svc,
-                                         self.info.runfolder).text
+                exec_string = self.get_exec_string()
```

Comment:
... and lazy properties for those that are expensive but cached:

https://github.com/Molmed/clarity-ext/blob/a5fa5bbe8e0bab3244598e285a4534bc4acddd95/clarity_ext/utils.py